assets/panel/panel.js

```diff
@@ -98,8 +98,9 @@ DebuggerPanel.prototype = {
     return { frames, selected };
   },
 
-  selectSource(sourceURL, sourceLine) {
-    this._actions.selectSourceURL(sourceURL, { location: { line: sourceLine }});
+  selectLocation(sourceOptions) {
```

Comment:
@lukaszsobek for now don't replace this function, as it is used in MC as well here 
https://dxr.mozilla.org/mozilla-central/source/devtools/client/shared/view-source.js#65.
Maybe just create a new `selectLocation` function and call `selectSource` from there.
That's probably the issue with the tests as well